includes/CAPx/Drupal/Processors/EntityProcessor.php

```diff
@@ -31,8 +31,176 @@ class EntityProcessor extends ProcessorAbstract {
    *   The new or updated wrapped entity.
    */
   public function execute($force = FALSE) {
+
+    // Set this force.
+    $this->force == $force;
+
+    try {
+      $multi = $this->getMapper()->getConfigSetting('multiple');
+    }
+    catch (\Exception $e) {
+      $multi = FALSE;
+    }
+
+    // Sometimes we need to create one, other times we need moar.
+    if (!empty($multi) && $multi == 1) {
+      $entity = $this->executeMultiple();
+    }
+    else {
```

Comment:
Multi is a setting from a boolean checkbox. It should never be anything that other than 1's and 0's. 

In any case, I do explicitly check for the value to be == 1. If it is anything else then go to the single option where it does not use the config setting for anything.